testing/test_version.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+from setuptools_scm.version import meta, simplified_semver_version
+
+
+@pytest.mark.parametrize('version, expected_next', [
+    pytest.param(meta("1.0.0"), "1.0.0", id='exact'),
+
+    pytest.param(meta("1.0"), "1.0.0", id='short_tag'),
```

Comment:
Does test imply that a tag of "1.0" generates a version of "1.0.0"? I'm unsure what the implications are of that. Is this an opt-in feature?

Reply:
this is opt-in, one has to specifiy simplified semver